content/blog/2017/08/2017-08-17-speaker-blog-blazemeter.adoc

```diff
@@ -0,0 +1,177 @@
++
+. Now create a Pipeline script. You can include all parts of
+your CI/CD process in this script: Commit, Build, Unit Test, Performance Test,
+etc., by creating different stages.
++
+This Pipeline has three stages: The first is called “build”. In this example it
+is empty, but you can add commands that will build your code. The second,
+called “Performance Tests”, creates a folder called “Taurus-Repo” and runs the
+Taurus script that we created. At the same time (note the “parallel” command),
+there is a “sleep” command for 60 seconds. Obviously it makes no sense to put
+those two commands together, this is just to show you the option of running 2
+commands in parallel. The third stage called “Deploy” is also empty in this
+example. This is where you could deploy your new version.
++
+[source, groovy]
+----
+node {
+   stage('Build') {
+      // Run the Taurus build
+   }
+   stage('Performance Tests') {
+    parallel(
+        BlazeMeterTest: {
+            dir ('Taurus-Repo') {
+                sh 'bzt <file_name>.yml -report'
+            }
+        }
+        , Analysis: {sleep 60})
+
+   }
+   stage(‘Deploy’) {
+   }
+}
+----
++
+image::/images/post-images/jenkinsworld2017/blazemeter/blazemeter-speaker-blog-2017-5.png[]
++
+Note that you can either add the Pipeline inline, or choose the “Pipeline
+script from SCM” option and add the URL to the script on GitHub (in this case
+you need to upload a `Jenkinsfile` to GitHub). With "Pipeline from SCM",
+whenever you need to update the tests, you can just add new commits to the
+`Jenkinsfile`.
++
+. Save the Pipeline
+. Click on ‘Build Now’ to run the Pipeline
++
+image::/images/post-images/jenkinsworld2017/blazemeter/blazemeter-speaker-blog-2017-6.png[]
++
+. Click on the new Build that is running now (build #6 in this example).
++
+image::/images/post-images/jenkinsworld2017/blazemeter/blazemeter-speaker-blog-2017-7.png[]
++
+. Click on ‘Console Output’ to see the test results:
++
+image::/images/post-images/jenkinsworld2017/blazemeter/blazemeter-speaker-blog-2017-8.png[]
++
+. In the Console Output you can see the test results and also the link to the report in BlazeMeter.
++
+image::/images/post-images/jenkinsworld2017/blazemeter/blazemeter-speaker-blog-2017-9.png[]
++
+image::/images/post-images/jenkinsworld2017/blazemeter/blazemeter-speaker-blog-2017-10.png[]
+
+
+That’s it! Jenkins Pipeline is now running open-source load testing tools via Taurus.
+
+Come to
+link:https://jenkinsworld20162017.sched.com/event/BYQn/learn-to-release-faster-by-load-testing-with-jenkins-free[my
+free hands-on workshop “Learn to Release Faster by Load Testing With Jenkins”]
+at Jenkins World 2017. You will learn how to test continuously with Jenkins,
+JMeter, BlazeMeter and Taurus, including how to  run JMeter with Jenkins, run
+the BlazeMeter plugin for Jenkins and how to use open-source Taurus. (Tue.,
+Aug. 29, 1:00pm - 5:00pm).
```

Comment:
Looks a bit weird to have this in parens after the regular test. Maybe integrate it into the first sentence, e.g. *at Jenkins World on Tuesday at 1 PM*?